### PR TITLE
[DOPS-895] Feature/ci update

### PR DESCRIPTION
**Pull request by Cre-eD**
# Task
[DOPS-895] Create new CI for new D3 repos.
## Changes
1. CI updated.

## Author
Signed-off-by: Dmitriy Creed <creed@soramitsu.co.jp>

[DOPS-895]: https://soramitsu.atlassian.net/browse/DOPS-895

### CODE REVIEW

File: deploy/docker-compose.eth.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ services:
       - d3-network
 
   d3-eth-deposit:
-    image: nexus.iroha.tech:19002/d3-deploy/eth-deposit:develop
+    image: nexus.iroha.tech:19002/soranet/eth-deposit:develop
```
Comment (Mingela) [inline deploy/docker-compose.eth.yml]: correct the port please?

File: deploy/docker-compose.eth.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ services:
       - d3-network
 
   d3-eth-testing:
-    image: nexus.iroha.tech:19002/d3-deploy/eth-testing-endpoints:develop
+    image: nexus.iroha.tech:19002/soranet/eth-testing-endpoints:develop
```
Comment (Mingela) [inline deploy/docker-compose.eth.yml]: same here

**Review by Mingela — APPROVED** [review]

**Review by kogeler — CHANGES_REQUESTED** [review]
Fix ports

**Review by ravenolf — COMMENTED** [review]
Check bagdan comment

File: Jenkinsfile [hunk 3]
```diff
@@ -7,30 +7,13 @@ pipeline {
     skipDefaultCheckout()
```
Comment (BulatSaif) [inline Jenkinsfile]:
```suggestion
```

File: Jenkinsfile [hunk 4]
```diff
@@ -7,30 +7,13 @@ pipeline {
...
-            builds.cancelSameJobBuilds()
-          }
-        }
-      }
-    }
     stage('Tests') {
-      agent { label 'd3-build-agent' }
       steps {
         script {
           def scmVars = checkout scm
```
Comment (BulatSaif) [inline Jenkinsfile]:
```suggestion
```
change `scmVars.` to `env.`

File: Jenkinsfile [hunk 5]
```diff
@@ -118,26 +101,24 @@ pipeline {
         }
       }
     }
-
     stage('Build and push docker images') {
-      agent { label 'd3-build-agent' }
       steps {
         script {
           def scmVars = checkout scm
```
Comment (BulatSaif) [inline Jenkinsfile]:
```suggestion
```

**Review by Mingela — APPROVED** [review]

**Review by ravenolf — APPROVED** [review]

**Review by BulatSaif — APPROVED** [review]

**Review by kogeler — APPROVED** [review]
OK

**Pull request merged**